controlplane/eks/api/v1beta1/awsmanagedcontrolplane_types.go

```diff
@@ -167,6 +168,10 @@ type AWSManagedControlPlaneSpec struct { //nolint: maligned
 	// +kubebuilder:default=false
 	DisableVPCCNI bool `json:"disableVPCCNI,omitempty"`
 
+	// VpcCni is used to set configuration options for the VPC CNI plugin
+	// +optional
+	VpcCni VpcCni `json:"VpcCni,omitempty"`
```

Comment:
@richardcase Ah I haven't noticed that but this should be lowercase.

I'll fix it in my PR. :)